sdk/storage/azblob/service.go

```diff
@@ -15,32 +15,78 @@ const (
 	scope = "https://storage.azure.com/.default"
 )
 
+// ServiceVersion enumerates the possible service API versions.
+type ServiceVersion string
+
+const (
+	// V2019_02_02 specifies the 2019-02-02 service API version.
+	V2019_02_02 = "2019-02-02"
+)
+
+//ServiceClient provides access to Azure Storage service resources and blob containers.
 type ServiceClient struct {
 	u *url.URL
 	p azcore.Pipeline
+	o ServiceClientOptions
```

Comment:
I assume this was mostly added due to ServiceVersion? Maybe we should just put that here and let the rest of the options be GC'd.